tsl/src/hypertable.c

```diff
@@ -223,7 +223,7 @@ update_replication_factor(Hypertable *const ht, const int32 replication_factor_i
 	if (list_length(ht->data_nodes) < replication_factor)
 		ereport(ERROR,
 				(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
```

Comment:
Also can be used `TS_INSUFFICIENT_NUM_DATA_NODES`, but likely current is more accurate